recipe_scrapers/owenhan.py

```diff
@@ -0,0 +1,22 @@
+from ._abstract import AbstractScraper
+
+
+class OwenHan(AbstractScraper):
+    @classmethod
+    def host(cls):
+        return "owen-han.com"
+
+    def author(self):
+        return "Owen Han"
```

Comment:
In general this pull request looks great, thanks @ephemer4l!

The only thing I worry about is this constant return value for the `author` method.

If Owen hosts recipes from other authors on the site then they wouldn't be attributed correctly - perhaps it'd be safer if we retrieve this field from the schema.org content in the HTML?

Reply:
Thank you. Do you think I should also get the image from the schema.org?

Reply:
Ah, good question - yep, that probably makes sense too :+1:

Reply:
Alas I can't get the title from the schema.org for some reason. I've tried with `self.schema.headline()` and `self.schema.title()` but neither seemed to work. Am I missing something here?

Reply:
Hmm, that's puzzling.  I'm not sure at the moment - I'll try to look into that in a bit more detail tomorrow before merging.  My guess is that it'll be something to do with the way that on-page metadata is handled (as seen in [this code file](https://github.com/hhursev/recipe-scrapers/blob/9ecaec334661b3d86ccad3df3abe30409a62d966/recipe_scrapers/_schemaorg.py)).